core-plugins/pom.xml

```diff
@@ -90,7 +90,6 @@
     <dependency>
       <groupId>org.python</groupId>
       <artifactId>jython-standalone</artifactId>
-      <version>2.5.2</version>
```

Comment:
We should inherit the version from the parent pom since there's no reason to duplicate or override the value.